### PR TITLE
Feature/vt 05 order workflow data model

### PR DESCRIPTION
**Pull request by vinhvoniteco**
vt-05: create order workflow repository

### CODE REVIEW

File: VirtoCommerce.OrderModule.Web/Controllers/Api/WorkflowController.cs [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+            _blobStorageProvider = blobStorageProvider;
+        }
+
+        [HttpGet]
+        [Route("{organizationId}")]
+        [ResponseType(typeof(Workflow))]
+        public IHttpActionResult Get(string organizationId)
+        {
+            var workflow = _workflowService.GetByOrganizationId(organizationId);
+            return Ok(new { data = workflow });
```
Comment (eMazeika) [inline VirtoCommerce.OrderModule.Web/Controllers/Api/WorkflowController.cs]: returned object must correspond the declared [ResponseType(typeof(Workflow))]
change 
return Ok(new { data = workflow });
to
return Ok(workflow);

File: VirtoCommerce.OrderModule.Web/Controllers/Api/WorkflowController.cs [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        }
+
+        [HttpPost]
+        [Route("")]
+        [ResponseType(typeof(Workflow))]
+        [CheckPermission(Permission = WorkflowPredefinedPermissions.Upload)]
+        public IHttpActionResult Upload([FromBody] Workflow model)
+        {
+            if (model == null)
+                return BadRequest();
```
Comment (eMazeika) [inline VirtoCommerce.OrderModule.Web/Controllers/Api/WorkflowController.cs]: throw new InvalidOperationException
with message

File: VirtoCommerce.OrderModule.Web/Controllers/Api/WorkflowController.cs [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+            var errorCode = ValidateWorkflowFile(model.JsonPath);
+            if (!string.IsNullOrEmpty(errorCode))
+            {
+                var err = new HttpError(errorCode);
```
Comment (eMazeika) [inline VirtoCommerce.OrderModule.Web/Controllers/Api/WorkflowController.cs]: throw new InvalidOperationException

**Review by eMazeika — CHANGES_REQUESTED** [review]

File: VirtoCommerce.OrderModule.Web/Controllers/Api/WorkflowController.cs (hunk 3 above)
Comment (tonhuynh) [reply VirtoCommerce.OrderModule.Web/Controllers/Api/WorkflowController.cs]: it's fixed

File: VirtoCommerce.OrderModule.Web/Controllers/Api/WorkflowController.cs (hunk 2 above)
Comment (tonhuynh) [reply VirtoCommerce.OrderModule.Web/Controllers/Api/WorkflowController.cs]: it's fixed

File: VirtoCommerce.OrderModule.Web/Controllers/Api/WorkflowController.cs (hunk 1 above)
Comment (tonhuynh) [reply VirtoCommerce.OrderModule.Web/Controllers/Api/WorkflowController.cs]: it's fixed

**Review by eMazeika — APPROVED** [review]

**Pull request merged**